datasets/piqa/README.md

```diff
@@ -20,9 +20,58 @@ task_ids:
 - multiple-choice-qa
 paperswithcode_id: piqa
 pretty_name: 'Physical Interaction: Question Answering'
+train-eval-index:
+- config: plain_text
+  task: text-generation
```

Comment:
text-generation is like autocomplete, to generate a new text we use text2text-generation
```suggestion
  task: text2text-generation
```